integration/app/test-demo/src/pages/VideoFilterTestApp.tsx

```diff
@@ -0,0 +1,109 @@
+  FormField,
+  Select,
+} from 'amazon-chime-sdk-component-library-react';
+import MeetingInfo from '../components/MeetingInfo';
+import MeetingLeaveControl from '../components/MeetingLeaveControl';
+import MeetingForm from '../containers/MeetingForm';
+
+export const VideoFilterTestApp: React.FC = () => {
+  const defaultOptions: BackgroundBlurOptions = {
+    blurStrength: 20,
+    filterCPUUtilization: 20,
+    reportingPeriodMillis: 1000,
+  };
+  const [backgroundBlurOptions, setBackgroundBlurOptions] = useState<BackgroundBlurOptions>(defaultOptions);
+
+  const setBlurStrength = (newBlurStrength: number): void => {
+    console.log(`Setting blur strength to ${newBlurStrength} - should re-initialize the bg blur provider`);
+    const options = {...backgroundBlurOptions, blurStrength : newBlurStrength};
+    setBackgroundBlurOptions(options);
+  }
+
+  return(
+    <div style={{ display: 'flex', flexDirection: 'column', width: '70%', margin: 'auto' }}>
+      <BackgroundBlurProvider options={backgroundBlurOptions}>
+        <MeetingProvider {...{logLevel: LogLevel.INFO}}>
+        <h3 data-testid='app-name'>Video Filter Test</h3>
+        <MeetingForm />
+        <Meeting onBlurStrengthChanged={(newBlurStrength) => setBlurStrength(newBlurStrength)}/>
+        </MeetingProvider>
+      </BackgroundBlurProvider>
+    </div>
+    )
+};
+
+interface Props {
+  onBlurStrengthChanged: (blurStrength: number) => void;
+}
+
+const Meeting: React.FC<Props> = ({ onBlurStrengthChanged }) => {
+  const { isBackgroundBlurSupported, createBackgroundBlurDevice } = useBackgroundBlur();
+  const meetingManager = useMeetingManager();
+  const { selectedDevice } = useVideoInputs();
+  const { toggleVideo } = useLocalVideo();
+  const meetingStatus = useMeetingStatus();
+  const [ blurStrength, setBlurStrength ] = useState<string>("20");
+
+  useEffect(() => {
+    const addBackgroundBlur = async () => {
+      if (
+        isBackgroundBlurSupported &&
+        meetingStatus === MeetingStatus.Succeeded
+      ) {
+        console.log("Creating background blur device called");
+        const chosenVideoTransformDevice = await createBackgroundBlurDevice(selectedDevice);
+        console.log(chosenVideoTransformDevice);
+        await meetingManager.selectVideoInputDevice(chosenVideoTransformDevice);
+        toggleVideo();
+      }
+    };
+    addBackgroundBlur();
+  }, [
+    isBackgroundBlurSupported,
+    meetingStatus,
+  ]);
+
+  const updateBlurStrength = (e: ChangeEvent<HTMLSelectElement>) => {
+    e.preventDefault();
+    setBlurStrength(e.target.value);
+    console.log('Updated to blurStrength:' + e.target.value);
+    const newBlurStrength = parseInt(e.target.value);
+    onBlurStrengthChanged(newBlurStrength);
+  }
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column', width: '70%', margin: 'auto' }}>
+    <MeetingInfo />
+    <div style={{height: '30rem', width:`100%` }}>
+      <VideoTileGrid />
+    </div>
+    <ControlBar
```

Comment:
Need to add some spacing between `VideoTileGrid` and `ControlBar`, current layout will overlay the top shadow of `ControlBar`.
![image](https://user-images.githubusercontent.com/20471438/154132883-01a06a9b-93ed-4c1f-b2d3-c7003880a6c1.png)